src/cf-layout/usage.md

```diff
@@ -1371,15 +1495,18 @@ The image should be 195px in height to conform to this standard.
         <div class="m-hero_text">
             <h1 class="m-hero_heading">Hero title</h1>
             <p class="m-hero_subhead">
-                Hero text goes here. This paragraph has a recommended maximum length of 185 characters.
-                This paragraph has a recommended maximum length of 185 characters.
+                Hero text goes here. This paragraph has a recommended maximum
+                length of 185 characters. This paragraph has a recommended
+                maximum length of 185 characters.
             </p>
             <a class="m-hero_cta" href="#">
                 Call to action
             </a>
         </div>
         <div class="m-hero_image-wrapper">
-            <div class="m-hero_image" style="background-image: url('http://www.consumerfinance.gov/static/fin-ed-resources/static/img/parents_hero_760x390.png')"></div>
+            <div class="m-hero_image"
+                 style="background-image: url('http://www.consumerfinance.gov/static/fin-ed-resources/static/img/parents_hero_760x390.png')">
```

Comment:
Yeah, they can and probably should be updated to this format: https://s3.amazonaws.com/files.consumerfinance.gov/f/images/PC_hero.original.jpg